streams/next.json

```diff
@@ -1,118 +1,118 @@
 {
     "stream": "next",
     "metadata": {
-        "last-modified": "2023-10-31T21:21:00Z",
-        "generator": "fedora-coreos-stream-generator v0.2.14"
+        "last-modified": "2023-11-03T03:36:29Z",
+        "generator": "fedora-coreos-stream-generator v0.2.14-1-g8ef5cb1"
```

Comment:
not sure what this is about (see the warning from Check stream diffs), but the rest of the diff looks good.

Reply:
Likely from https://github.com/coreos/fedora-coreos-stream-generator/pull/61 which added a commit.